test/unit/block_test.rb

```diff
@@ -31,6 +31,10 @@ def test_op_write_raw_w
     assert_equal(source, template.render!)
   end
 
+  def test_invalid_blocks
```

Comment:
It seems like this is testing an invalid tag.  Also, it seems like the test belongs in test/unit/tokenizer_test.rb, since the code being tested is in the tokenizer.